hack/schemas/testdata/inline-anyof/output.json

```diff
@@ -0,0 +1,92 @@
+{
+  "type": "object",
+  "anyOf": [
+    {
+      "$ref": "#/definitions/SkaffoldPipeline"
+    }
+  ],
+  "$schema": "http://json-schema-org/draft-07/schema#",
+  "definitions": {
+    "InlineOneOfStruct": {
```

Comment:
So now along with the possibilities of `TestStruct`, we are generating the Schema for InlineOneofStruct?

Reply:
question asked in #1908, thank you!